ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -1192,6 +1200,62 @@ func TestIsZeroWhenConfigHasValues(t *testing.T) {
 	}
 }
 
+func TestConvertToTaskDefinitionWithECSParams_HealthCheckOverrideCompose(t *testing.T) {
+	containerConfig := &adapter.ContainerConfig{
+		Name:  "web",
+		Image: "httpd",
+		HealthCheck: &ecs.HealthCheck{
+			Command:     aws.StringSlice([]string{"CMD", "curl", "-f", "http://example.com"}),
+			Interval:    aws.Int64(int64(91)),
+			Timeout:     aws.Int64(int64(17)),
+			Retries:     aws.Int64(int64(7)),
+			StartPeriod: aws.Int64(int64(73)),
+		},
+	}
+
+	ecsParamsString := `version: 1
+task_definition:
+  services:
+    web:
+      healthcheck:
+        test: curl -f http://localhost
+        interval: 10m
+        timeout: 15s
+        retries: 5
+        start_period: 50`
+
+	content := []byte(ecsParamsString)
+
+	tmpfile, err := ioutil.TempFile("", "ecs-params")
+	assert.NoError(t, err, "Could not create ecs params tempfile")
+
+	defer os.Remove(tmpfile.Name())
+
+	_, err = tmpfile.Write(content)
+	assert.NoError(t, err, "Could not write data to ecs params tempfile")
+
+	err = tmpfile.Close()
+	assert.NoError(t, err, "Could not close tempfile")
+
+	ecsParamsFileName := tmpfile.Name()
+	ecsParams, err := ReadECSParams(ecsParamsFileName)
+	assert.NoError(t, err, "Could not read ECS Params file")
+
+	containerConfigs := []adapter.ContainerConfig{*containerConfig}
+	taskDefinition, err := convertToTaskDefWithEcsParamsInTest(t, containerConfigs, "", ecsParams)
+
+	containerDefs := taskDefinition.ContainerDefinitions
+	web := findContainerByName("web", containerDefs)
+
+	if assert.NoError(t, err) {
+		assert.Equal(t, []string{"CMD-SHELL", "curl -f http://localhost"}, aws.StringValueSlice(web.HealthCheck.Command))
+		assert.Equal(t, aws.Int64(600), web.HealthCheck.Interval)
+		assert.Equal(t, aws.Int64(15), web.HealthCheck.Timeout)
+		assert.Equal(t, aws.Int64(5), web.HealthCheck.Retries)
+		assert.Equal(t, aws.Int64(50), web.HealthCheck.StartPeriod)
+	}
+}
+
```

Comment:
I was wondering if we should consider validating that the `HealthCheck.Command` is not blank, since it's the only required field for HealthCheck. We should be validating in the converter that not both `test` and `command` are specified in the `ecs-params.yml` I think, so maybe adding in this extra validation might also be a good idea?